x-pack/plugins/ml/public/application/routing/routes/new_job/job_type.tsx

```diff
@@ -34,18 +33,18 @@ export const jobTypeRouteFactory = (navigateToPath: NavigateToPath, basePath: st
   ],
 });
 
-const PageWrapper: FC<PageProps> = ({ location, deps }) => {
-  const { index, savedSearchId }: Record<string, any> = parse(location.search, { sort: false });
-  const { context } = useResolver(
-    index,
-    savedSearchId,
-    deps.config,
-    deps.dataViewsContract,
-    basicResolvers(deps)
-  );
+const PageWrapper: FC = () => {
+  const { context } = useRouteResolver('full', ['canGetJobs'], {
+    getMlNodeCount,
```

Comment:
Can we use basicResolvers() here?

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75

Reply:
Updated in https://github.com/elastic/kibana/pull/159176/commits/018c48948231395246006974d3ccf25a061eef75